.github/workflows/plugin_update.yml

```
@@ -0,0 +1,70 @@
+name: Plugins Update file test
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # run daily at midnight
+  push:
+    branches:
+      - '*'
+
+jobs:
+  check-plugins:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      
+      - name: Configure git & gh
+        run: |
+          
+          echo ${{ secrets.GH_TOKEN }} | gh auth login --with-token
+          gh auth status
+          git config --global user.email "ash.191245141@gmail.com"
+          git config --global user.name "ash-sxn"
+
+      - name: Set branch Names and start the containers
+        run: |
+          # Set the name of the branch to create the pull request from
+          BRANCH_NAME="update-plugins-$(date +'%Y/%m/%d/%H/%M/%S')"
```

I think should share this with GitHub Action environment variable right?

```suggestion
          echo "BRANCH_NAME=update-plugins-$(date +'%Y/%m/%d/%H/%M/%S')" >> $GITHUB_ENV
```

CMIIW